prime-router/metadata/HL7/catchall/hl7/message/ORU_R01.yml

```diff
@@ -45,7 +45,7 @@ resources:
     resourcePath: segments/PV1/Encounter
     isReferenced: true
     additionalSegments:
-      - .PATIENT.VISIT.PV2
+      - .VISIT.PV2
```

Comment:
Can we add a test to cover this?  This does seem incorrect and is not covered by the current mapping cases

Reply:
Dug into this and it appears this isn't technically incorrect, it just against how we are listing out our segments. So no test needed, and if this had been technically incorrect, we do have pv2 data in the oru test and it would have failed.

Longer story: 
It looks like the converter library is very forgiving in how you list out your segments, so long as they have a least one part of the segment group trail. For example the following worked (note the absence of a `group` specifier:
```
  - resourceName: Encounter
    segment: PATIENT_RESULT.PATIENT.VISIT.PV1
    resourcePath: segments/PV1/Encounter
    isReferenced: true
    additionalSegments:
      - .VISIT.PV2
      - MSH
  ```

But if you modify additionalSegments like so, it will not be able to use pv2 in the mappings:
```
    additionalSegments:
      - PV2
      - MSH
```
I assume the reasoning behind this is that if they can tell what group the segment belongs in, they will use it, otherwise they won't use it because segments can belong to different groups.